src/Asana/Dispatcher/OAuthDispatcher.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Asana\Dispatcher;
 
-use \OAuth2;
+use OAuth2;
```

Comment:
I thought there was a reason for the leading backslash, like it is explicitly starting from the root namespace. Do we not need this?